core/schema.py

```diff
@@ -0,0 +1,9 @@
+import graphene
+
+import app.api.schema
```

Comment:
Could we use `from app.api.schema import Query` here instead. It'll potentially reduce the length of arguments on line 6. Except, of course, there's a specific and valid reason you chose to import the class this way.